app/controllers/concerns/navigation.rb

```diff
@@ -11,7 +11,7 @@ module Navigation
   attr_reader :active_categories
 
   def assign_active_categories(*categories)
-    categories.each do |category|
+    categories.map do |category|
```

Comment:
Not too sure myself, this was originally done by @dougdroper. Looks like he wanted `assign_active_categories` to explicitly return a value?